src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -358,11 +359,15 @@ export class DBTCoreProjectIntegration
           if (matches) {
             compiled_stmt = matches[1].trim();
           }
+          return { ...result, compiled_stmt };
         } catch (err) {
-          console.error("error while executing querytemplate conversion", err);
+          const message = `Error while executing sql: ${compiledQuery}`;
+          this.dbtTerminal.error("dbtCore:executeSQL", message, err);
+          if (err instanceof PythonException) {
+            throw new ExecuteSQLError(err.exception.message, compiledQuery!);
+          }
+          throw new ExecuteSQLError((err as Error).message, compiledQuery!);
```

Comment:
The implementation has changed. Before exception on query template was ignored. 

I think the exception handling should be split in two, one for actual exectuion and one for the query template replacement.

Reply:
@AdiGajbhiye Can you please address this comment?

Reply:
`const matches = queryRegex.exec(result.compiled_sql);`

If above line fails it will throw an exception, but we are not really interested in it as we can just use the `compiledStatement`. That's what I meant that the code changes you made are not the same as it used to be. Also the code that is tried can never throw a PythonExxception.